src/patternfly/demos/Card/examples/CardView.md

```diff
@@ -284,3 +287,1028 @@ wrapperTag: div
+        {{#> accordion-toggle-text}}
+          <span class="pf-u-font-weight-light">
+            CPU 1
+          </span>
+        {{/accordion-toggle-text}}
+        {{#> accordion-toggle-icon}}{{/accordion-toggle-icon}}
+      {{/accordion-toggle}}
+      {{#> accordion-expanded-content accordion-expanded-content--IsExpanded="true"}}
+        {{#> accordion-expanded-content-body}}
+          {{> cardview-demo--stacked-sparklines}}
+        {{/accordion-expanded-content-body}}
+      {{/accordion-expanded-content}}
+
+      {{#> accordion-toggle accordion-toggle--attribute='aria-expanded="false"'}}
+        {{#> accordion-toggle-text}}
+          <span class="pf-u-font-weight-light">
+            CPU 2
+          </span>
+        {{/accordion-toggle-text}}
+        {{#> accordion-toggle-icon}}{{/accordion-toggle-icon}}
+      {{/accordion-toggle}}
+      {{#> accordion-expanded-content}}
+        {{> cardview-demo--stacked-sparklines}}
+      {{/accordion-expanded-content}}
+
+      {{#> accordion-toggle accordion-toggle--attribute='aria-expanded="false"'}}
+        {{#> accordion-toggle-text}}
+          <span class="pf-u-font-weight-light">
+            CPU 3
+          </span>
+        {{/accordion-toggle-text}}
+        {{#> accordion-toggle-icon}}{{/accordion-toggle-icon}}
+      {{/accordion-toggle}}
+      {{#> accordion-expanded-content}}
+        {{> cardview-demo--stacked-sparklines}}
+      {{/accordion-expanded-content}}
+    {{/accordion}}
+  {{/card-body}}
+{{/card}}
+```
+
+### Trend card 1
+```hbs
+{{#> gallery gallery--modifier="pf-m-gutter" gallery--attribute='style="--pf-l-gallery--GridTemplateColumns--min: 360px;"'}}
+  {{#> card card--id="trend-card-1-example"}}
+    {{#> card-header}}
+      {{#> l-flex l-flex--modifier="pf-m-column pf-m-space-items-none"}}
+        {{#> card-title card-title--attribute=(concat 'id="' card--id '-title"')}}
+          {{#> title title--modifier="pf-m-2xl"}}
+            1,050,765 IOPS
+          {{/title}}
+        {{/card-title}}
+        <span class="pf-u-color-200">
+          Workload
+        </span>
+      {{/l-flex}}
+      {{#> card-actions}}
+        {{> dropdown dropdown--id=(concat card--id '-dropdown') dropdown-toggle--text="Filter" dropdown-menu--modifier="pf-m-align-right"}}
+      {{/card-actions}}
+    {{/card-header}}
+    {{#> card-body}}
+      {{> cardview-demo--chart cardview-demo--chart--IsChart2="true"}}
+    {{/card-body}}
+  {{/card}}
+{{/gallery}}
+```
+
+### Trend card 2
+```hbs
+{{#> gallery gallery--modifier="pf-m-gutter" gallery--attribute='style="--pf-l-gallery--GridTemplateColumns--min: 360px;"'}}
+  {{#> card card--id="trend-card-2-example"}}
+    {{#> card-header}}
+      {{#> l-flex l-flex--modifier="pf-m-align-items-center"}}
+        {{#> l-flex-item l-flex-item--modifier="pf-m-flex-none"}}
+          {{#> l-flex l-flex--modifier="pf-m-column pf-m-space-items-none"}}
```

Comment:
Looks like there are a few more places we use `.pf-l-flex.pf-m-column.pf-m-space-items-none` in this demo that could just be a `<div>` with `<div>` children. It's fine to leave them as flex layouts if you want, just seems a little heavy handed to stack block elements.